### DISCUSSION TITLE
AppStore [Apr 16]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5527] [Discord Threads]
> 🙋‍♂️ **out-of-matrix. io**  
#### the docs are showing this but I don't have the `App store` on my end  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1085189598499590235/image.png)  
![image1](https://cdn.discordapp.com/attachments/661905455894888493/1085189598898036866/image.png)

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### Menu is now moved under "Accounts";    
#### - open `Project Menu`   
#### - Click on `Accounts`   
#### - Under `Account Settings`, click on `App Store`   
####    
#### If you do not see this menu, you do not have admin permissions  
#  
> 🙋‍♂️ **CrowZ**  
#### <@134344103726022656> are u referring something similiar like this??  
#  
> 🙋‍♂️ **Isma**  
#### Exactly!! Thank you ❤️  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1085189599149686904)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
